testthat_py36.R

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env Rscript
+# Usage: testthat.R [TEST_FILE]
+#
+# TEST_FILE Run tests for the specified TEST_FILE (e.g. 'test_keras.R').
+#
+library("testthat")
+
+args = commandArgs(trailingOnly=TRUE)
+
+check_gpu <- function() {
+  if (Sys.getenv("CUDA_VERSION") == "") {
+    skip("Skipping GPU tests for CPU image")
+  }
+}
+
+if (length(args)==1) {
```

Comment:
Could you add a comment on why this is necessary?

Reply:
Just mirroring the same from the standard tests file (https://github.com/Kaggle/docker-rstats/blob/691037f4a3e6e08fba08b9c6d0303a8f726477c2/testthat.R#L16). Isn't needed at the moment.